### DISCUSSION TITLE
Please provide Ubuntu 24.04 packages

### DISCUSSION THREAD

**Discussion by bastelfreak** [puppetlabs/pdk#1331] [Feature Request] [answered]
## Use Case

As an Ubuntu user I need packages for my distro. Ubuntu 24.04 will have the stable release soon and there are already images available.

## Describe the Solution You Would Like

Packages for Ubuntu 24.04

**Comment by david22swan** [comment, marked as answer]
In regards to Ubuntu 24.04 support we likely be adding shortly after support for it has been released for the Agent.
